app/Server.hs

```diff
@@ -9,10 +9,9 @@ responses.
 module Server
     (runServer) where
 
-import Config (markdownPath, serverConf)
+import Config (serverConf)
```

Comment:
This is great; you can check whether `markdownPath` is used anywhere else, and (assuming not), remove it entirely from `Config` and the configuration YAML file.